bumpversion.sh

```diff
@@ -30,12 +30,12 @@ fi
 
 if [[ $2 == '--tag' ]]; then
   if git branch --contains $(git rev-parse --verify HEAD) | grep -E 'main'; then
-    bumpversion --tag --commit $1
```

Comment:
Did you test that bumpversion.sh works with these changes? Since it won't show up in unit tests